dashboards/lodestar_debug_gossipsub.json

```diff
@@ -1,58 +1,12 @@
 {
-  "__inputs": [
```

Comment:
should this `__inputs` header be removed? I think if you do `Export for sharing externally` it is added

Reply:
I'm not sure, not related to this change so I reverted